pkg/cmdimpl/add.go

```diff
@@ -238,25 +245,32 @@ func generateHelmManifest() ([]byte, error) {
 	return fluxops.CallFlux(cmd)
 }
 
-func getOwner() string {
+func getOwner() (string, error) {
 	owner, err := fluxops.GetOwnerFromEnv()
 	if err != nil || owner == "" {
-		owner = getOwnerFromUrl(params.Url)
+		owner, err = getOwnerFromUrl(params.Url)
+		if err != nil {
+			return "", fmt.Errorf("could not getting owner %s", err)
```

Comment:
This should probably say either `could not get owner` or ` unable to retrieve owner`